tox.ini

```diff
@@ -37,6 +37,7 @@ passenv =
     HOME
 extras =
     mlflow
+    opt_einsum
```

Comment:
@cthoyt , I think this should make sure the code is tested *with* `opt_einsum`

Reply:
yes that's correct